### ISSUE TITLE
Add rule for leading/trailing whitespace

### ISSUE THREAD

**Issue by jdforrester** [wikimedia/banana-checker#89]
Raised at https://gerrit.wikimedia.org/r/c/mediawiki/core/+/657539

**Comment by Nikerabbit** [comment]
Leading is not as problematic (but still probably good idea to discourage as it's indication of lego messages). Trailing whitespace will be stripped in translatewiki.net and will cause issues if it is required in translations, as it cannot be added there.

**Comment by jdforrester** [comment]
> Leading is not as problematic (but still probably good idea to discourage as it's indication of lego messages).

Yeah; mostly I thought it was worth it just for completeness alone, but spotting lego messages is a good thing too.

> Trailing whitespace will be stripped in translatewiki.net and will cause issues if it is required in translations, as it cannot be added there.

Right; we'll make it default, once it's working. ;-)
